glossary.md

```diff
@@ -10,51 +10,49 @@ This page contains a selection of key terms important to understanding the struc
+The technique used for measuring a statistical variable. Describes how a measurement is made, whether by count or estimate or some other approach. May name the group making the measurement to indicate a certain organizational method of measurement is used. Examples include [the American Community Survey](https://datacommons.org/browser/dc/gg17432) and [`WorldHealthOrganizationEstimates`](https://datacommons.org/browser/WorldHealthOrganizationEstimates). Multiple measurement methods may be specified for any given node.
 
-- **[Observation period](https://datacommons.org/browser/observationPeriod)**
+### **[Observation Period](https://datacommons.org/browser/observationPeriod)**
 
-The time period over which an observation is made. Specified in [ISO 8601 formatting for durations](https://en.wikipedia.org/wiki/ISO_8601#Durations). A complete list of properties can be found at <https://datacommons.org/browser/observationPeriod>.
+The time period over which an observation is made. Specified in [ISO 8601 formatting for durations](https://en.wikipedia.org/wiki/ISO_8601#Durations).
 
-- **[Scaling factor](https://datacommons.org/browser/scalingFactor)**
+### **[Scaling Factor](https://datacommons.org/browser/scalingFactor)**
 
 Property of statistical variables that measure proportions, used in conjunction with the measurementDenominator property to indicate the multiplication factor applied to the proportion's denominator (with the measurement value as the final result of the multiplication) when the numerator and denominator are not equal.
 
-As an example, in 1999, [approximately 36% of Canadians were Internet users](https://datacommons.org/browser/dc/o/2mthzyv99kd73). Here the measured value of `Count_Person_IsInternetUser_PerCapita` is 36, and the scaling factor or denominator for this per capita measurement is 100. Without the scaling factor, we would interpret the value to be 36/1, or 3600%.
+As an example, in 1999, [approximately 36% of Canadians were Internet users](https://datacommons.org/browser/dc/o/0d9e3dd3y6yt3). Here the measured value of `Count_Person_IsInternetUser_PerCapita` is 36, and the scaling factor or denominator for this per capita measurement is 100. Without the scaling factor, we would interpret the value to be 36/1, or 3600%.
 
-A complete list of properties can be found at <https://datacommons.org/browser/scalingFactor>.
```

Comment:
please keep this line, even if redundant. the headings don't look very clickable at the moment.

Reply:
Added this back to be consistent with other items